docs/sdd/design/dev-tool-settings/components/encryption-service.md

```
@@ -27,68 +27,67 @@
 
 ### 公開API/メソッド
 
-#### `encrypt(plaintext: string): Promise<EncryptedData>`
+#### `encrypt(plainText: string): Promise<string>`
 
-**説明**: SSH秘密鍵を暗号化
+**説明**: SSH秘密鍵を暗号化し、`iv:authTag:encrypted` 形式の文字列を返す
 
 **パラメータ**:
 | 名前 | 型 | 必須 | 説明 |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)